### PR TITLE
Minor copy edits and some grammar corrections

### PR DESCRIPTION
**Pull request by avive**
@zalmen @gavraz plz review

### CODE REVIEW

File: hare/README.md [hunk 1]
```diff
@@ -3,36 +3,55 @@
...
-`f` - the number of active dishonest participants. <br />
-`Pi` - A participant in the consensus. <br />
-`Si` - The current set blocks observed by Pi. <br />
+`N` - The number of active participants.
+
+`f` - The number of active dishonest participants.
+
+`Pi` - A participant in the consensus.
+
+`Si` - The current set blocks observed by Pi.
```
Comment (gavraz) [inline hare/README.md]: Add "at the beginning that is the initial set".

File: hare/README.md [hunk 2]
```diff
@@ -3,36 +3,55 @@
...
-* `Active` - Also acts as a speaker and share his knowledge.
-* `Leader` - a participant who can give a proposition to the network. There is only one leader.
+#### Roles
+During the BA, at each round, a party can be assigned with one of the following roles:
+* `Passive` - Listens and follows the protocol. This is the default role of a participant meaning every protocol participant is at least passive.
+* `Active` - Also acts as a speaker and share his knowledge
+* `Leader` - a participant who can give a proposition to the network. There is only one leader
 
+<<<<<<< HEAD
+#### Virtual Participation
```
Comment (gavraz) [inline hare/README.md]: Should be removed

File: hare/README.md [hunk 3]
```diff
@@ -3,36 +3,55 @@
...
+Virtual participation is an optimization made to ease a participant’s role. After a participant terminates, he sends a notify message declaring his commitment to his corresponding set. From here on out, all other participants can refer to his notify message as if it was sent in the current round.
+
+#### Certificate
+A proof to witnessing f+1 commits on a set S.
+=======
 **Certificate** - a proof to witnessing f+1 commits on a set S.
+>>>>>>> 209247fe5a6c409d263c35f272a853450573e1e8
+
+#### Safe Value Proof (SVP)
```
Comment (gavraz) [inline hare/README.md]: You can leave it as it is. But, I will change it later as the protocol changed and it is not accurate.

File: hare/README.md (hunk 1 above)
Comment (avive) [reply hare/README.md]: Add to pi or to si?

**Pull request merged**